### PR TITLE
chore: automatic tidy

### PR DESCRIPTION
**Pull request by marcoscaceres**
This workflow will automatically tidy up documents _after_ a pull request is merged, and it sends a pull request if was fixed by tidy. 

This allows folks to make contributions without needing to worry *too much* about the markup they are writing.

@yoavweiss, 99% tidy will do the right thing. However, if a contribution has garbage HTML (e..g, `<section>` is not closed), tidy will try to recover. In such cases, please double check that tidy has done what we want. 

Any issues, just ping!

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/pull/118.html" title="Last updated on Jun 7, 2021, 5:51 AM UTC (c8d0473)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/118/94f3eb5...c8d0473.html" title="Last updated on Jun 7, 2021, 5:51 AM UTC (c8d0473)">Diff</a>

### CODE REVIEW

**Comment by yoavweiss** [comment]
Thanks! This makes sense and will be nice to propagate. One nit: Can you also change tidyconfig to `indent: yes`? Makes things significantly nicer.

**Comment by marcoscaceres** [comment]
oh! that's way better indeed... I was wondering why it was looking kinda off.

**Review by yoavweiss — APPROVED** [review]
LGTM!!

File: index.html [hunk 1]
```diff
@@ -48,106 +50,142 @@
...
+        and adjustment of the system clock. The value of time may not always be
+        monotonically increasing and subsequent values may either decrease or
+        remain the same.
+      </p>
+      <p>
+        For example, the following script may record a positive number,
+        negative number, or zero for computed <code>duration</code>:
+      </p>
+      <pre class="example js">
+&gt;&gt;&gt;&gt;&gt;&gt;&gt; 5b1777c (use indent: yes)
```
Comment (foolip) [inline index.html:91]: @marcoscaceres this was not intentional, right? It looks like a conflict marker in https://w3c.github.io/hr-time/#example-1 now...

Comment (marcoscaceres) [reply index.html:91]: Whoa, woopsy!

Comment (marcoscaceres) [reply index.html:91]: Sent https://github.com/w3c/hr-time/pull/122

**Pull request merged**